Mathlib/GroupTheory/SpecificGroups/KleinFour.lean

```diff
@@ -61,7 +61,7 @@ theorem card : Fintype.card KleinFour = 4 :=
 
 /-- Klein four-group is a group of order 4. -/
 theorem nat_card : Nat.card KleinFour = 4 := by
-  simp only [Nat.card_eq_fintype_card]
+  simp (config := {decide := true}) only [Nat.card_eq_fintype_card]
```

Comment:
And maybe mention this simp config as well?

Reply:
Done.